<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1804</width>
    <height>1136</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RespiratoryCare</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="tabletTracking">
    <bool>true</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget{
background-image: url(:/1/5586989.jpg);
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_9">
    <item row="3" column="1">
     <widget class="QWidget" name="widget" native="true">
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>400</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient((-225deg, #3D4E81 0%, #5753C9 48%, #6E7FF3 100%));</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="styleSheet">
          <string notr="true">font: 18pt &quot;幼圆&quot;;</string>
         </property>
         <property name="title">
          <string>读入文件</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="5" column="0">
           <widget class="QPushButton" name="pushButton">
            <property name="styleSheet">
             <string notr="true">font: 18pt &quot;幼圆&quot;;
background-color:rgb(252, 239, 226)
</string>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QProgressBar" name="progressBar">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QProgressBar {
border: 2px solid grey;
border-radius: 5px;
background-color: #f6f6f6;
}

QProgressBar::chunk {
background-color: #ffaaa6;
width: 20px;
}

QProgressBar {
border: 2px solid grey;
border-radius: 5px;
text-align: center;
}
</string>
            </property>
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="3">
           <widget class="QPushButton" name="openButton">
            <property name="styleSheet">
             <string notr="true">font: 18pt &quot;幼圆&quot;;
background-color:rgb(252, 239, 226)
</string>
            </property>
            <property name="text">
             <string>打开文件</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>00:00</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0" colspan="3">
           <widget class="QPushButton" name="analysis">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color:rgb(252, 239, 226)
</string>
            </property>
            <property name="text">
             <string>分析信号</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QPushButton" name="play">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 18pt &quot;幼圆&quot;;
background-color:rgb(252, 239, 226)
</string>
            </property>
            <property name="text">
             <string>播放/暂停</string>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QPushButton" name="pushButton_2">
            <property name="styleSheet">
             <string notr="true">font: 18pt &quot;幼圆&quot;;
background-color:rgb(252, 239, 226)
</string>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="label_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>00:00</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSlider" name="horizontalSlider">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QWidget" name="widget_2" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>400</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;幼圆&quot;;</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>分析结果：</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QTextBrowser" name="textBrowser"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" rowspan="2" colspan="3">
     <widget class="QWidget" name="widget_3" native="true">
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0" colspan="7">
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="currentIndex">
          <number>4</number>
         </property>
         <widget class="QWidget" name="page">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QGraphicsView" name="graphicsView"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_2">
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QGraphicsView" name="graphicsView_2"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_3">
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QGraphicsView" name="graphicsView_3"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_4">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0" colspan="4">
            <widget class="QGraphicsView" name="graphicsView_4">
             <property name="styleSheet">
              <string notr="true">font: 14pt &quot;Adobe Heiti Std&quot;;</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_5">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QGraphicsView" name="graphicsView_5"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_6">
          <layout class="QGridLayout" name="gridLayout_11">
           <item row="0" column="0">
            <widget class="QGraphicsView" name="graphicsView_6"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_7">
          <layout class="QGridLayout" name="gridLayout_10">
           <item row="0" column="0">
            <widget class="QGraphicsView" name="graphicsView_7"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="timeButton">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 18pt &quot;幼圆&quot;;
background-color:rgb(252, 239, 226)
</string>
         </property>
         <property name="text">
          <string>波形图</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="spectrumButton">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 18pt &quot;幼圆&quot;;
background-color:rgb(252, 239, 226)
</string>
         </property>
         <property name="text">
          <string> 频谱图</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="spectrogramButton">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 18pt &quot;幼圆&quot;;
background-color:rgb(252, 239, 226)
</string>
         </property>
         <property name="text">
          <string>时频图</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="mfccButton">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 18pt &quot;幼圆&quot;;
background-color:rgb(252, 239, 226)
</string>
         </property>
         <property name="text">
          <string>MFCC频谱图</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QPushButton" name="melspectroButton">
         <property name="styleSheet">
          <string notr="true">font: 18pt &quot;幼圆&quot;;
background-color:rgb(252, 239, 226)</string>
         </property>
         <property name="text">
          <string>梅尔尺度的频谱图</string>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QPushButton" name="contrastButton">
         <property name="styleSheet">
          <string notr="true">font: 18pt &quot;幼圆&quot;;
background-color:rgb(252, 239, 226)</string>
         </property>
         <property name="text">
          <string>光谱对比度</string>
         </property>
        </widget>
       </item>
       <item row="1" column="6">
        <widget class="QPushButton" name="tonnetzButton">
         <property name="styleSheet">
          <string notr="true">font: 18pt &quot;幼圆&quot;;
background-color:rgb(252, 239, 226)</string>
         </property>
         <property name="text">
          <string>色调质心</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1804</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="bg.qrc"/>
 </resources>
 <connections/>
</ui>
